.github/workflows/sync_labels.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow synchronizes groups of labels that correspond
+# to items of selection list in Trac. It controls that in each
+# such case there is just one label of the list present.
+# Furthermore in the case of the state it checks the labels
+# to coincide with the corresponding review state.
+
+name: Synchronize selection list lables
+
+on:
+  pull_request_review:
+    types: [submitted]
+  issues:
+    types: [opened, reopened, closed, labeled, unlabeled]
+  pull_request:
```

Comment:
This needs to be `pull_request_target` since otherwise you don't have access to secrets.

Reply:
Thanks for the hint. In my tests I tested `commented` (since the two other `rev_states` were not possible) which worked well with `pull_request`.

Reply:
I add the event in addition.